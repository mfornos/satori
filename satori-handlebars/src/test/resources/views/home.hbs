{{#partial "title"}} Home {{/partial}}
{{#partial "content"}} content here {{/partial}}
{{> base}}