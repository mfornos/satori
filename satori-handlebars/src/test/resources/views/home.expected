

<!-- base.hbs -->
<html>
<head>
  <title>
     Home 
  </title>
</head>
<body>
  Lorem ipsum content here 
  footer</body>
</html>